src/Microsoft.DotNet.Build.Tasks/Targets/resources.targets

```diff
@@ -1,59 +1,38 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Don't think a targets file should have a DefaultTargets attribute.  Also is there a reason to change ToolsVersion?